src/tabs/presets/DetailedDialog/PresetsDetailedDialog.js

```diff
@@ -125,22 +125,57 @@ class PresetsDetailedDialog {
 
     _createOptionsSelect(options) {
         options.forEach(option => {
-            let selectedString = "selected=\"selected\"";
-            if (!option.checked) {
-                selectedString = "";
+            if (!option.childs) {
+                this._addOption(this._domOptionsSelect, option, false);
+            } else {
+                this._addOptionGroup(this._domOptionsSelect, option);
             }
-
-            this._domOptionsSelect.append(`<option value="${option.name}" ${selectedString}>${option.name}</option>`);
         });
 
         this._domOptionsSelect.multipleSelect({
-            placeholder: i18n.getMessage("dropDownAll"),
+            placeholder: i18n.getMessage("presetsOptionsPlaceholder"),
             formatSelectAll () { return i18n.getMessage("dropDownSelectAll"); },
             formatAllSelected() { return i18n.getMessage("dropDownAll"); },
             onClick: () => this._optionsSelectionChanged(),
             onCheckAll: () => this._optionsSelectionChanged(),
             onUncheckAll: () => this._optionsSelectionChanged(),
+            hideOptgroupCheckboxes: true,
+            singleRadio: true,
+            selectAll: false,
+            styler: function (row) {
+                let style = "";
+                if (row.type === 'optgroup') {
+                    style = 'font-weight: bold;';
+                } else if (row.classes.includes("optionHasParent")) {
+                    style = 'padding-left: 22px;';
+                }
+                return style;
+            },
+        });
+    }
+
+    _addOptionGroup(parentElement, optionGroup) {
+        const optionGroupElement = $(`<optgroup label="${optionGroup.name}"></optgroup>`);
+
+        optionGroup.childs.forEach(option => {
+            this._addOption(optionGroupElement, option, true);
         });
+
+        parentElement.append(optionGroupElement);
+    }
+
+    _addOption(parentElement, option, hasParent) {
```

Comment:
Can we have an OTHER category for this type?

Reply:
not sure what you mean here?

Reply:
![image](https://user-images.githubusercontent.com/8344830/149646182-e81fd442-adcc-40e8-b93e-c25a624db3e0.png)

Reply:
oh yeah i see. I figured we will give the authors the flexibility. If they want OTHER group - they can create it. Or they can leave checkboxes without group. They even can even do something like that if thats what they want:
![image](https://user-images.githubusercontent.com/2925027/149648107-6a091f2f-431b-4b5d-9c95-6d7baca2e336.png)

Reply:
Yes I figured that once the PR is merged, authors with lots of options would organise them under 'headings' to make it less of a mess.

Reply:
Awesome!